### PR TITLE
Omniwallet mobile app first commit

### PR DESCRIPTION
**Pull request by mobilecheng**
Omniwallet mobile app first commit by Kevin Zhang (mobilecheng)

### CODE REVIEW

File: lib/view/backupwallet/backup_wallet_index.dart [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  Widget build(BuildContext context) {
+    Widget pageContent(){
+      return Column(
+        children: <Widget>[
+          Container(
+            margin: EdgeInsets.only(top: 30,left: 20,right: 20),
+            child: CachedNetworkImage(
+              placeholder: (context, url) => new CircularProgressIndicator(),
+              errorWidget: (context, url, error) => new Icon(Icons.error),
+              imageUrl: 'http://www.pptbz.com/pptpic/UploadFiles_6909/201203/2012031220134655.jpg',
```
Comment (achamely) [inline lib/view/backupwallet/backup_wallet_index.dart]: This url does not resolve, is it supposed to?
If so, can you double check or include the asset locally?

File: lib/l10n/WalletLocalizations.dart [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      'welcomePageThreeContentFour' : "Do not open emails or links from unknown senders.",
+      'welcomePageThreeContentFive' : "Regularly update your browser and operating system.",
+      'welcomePageThreeContentSix' : "Use official security software. Do not install "
+          "unknown software which could be hacked.",
+      'welcomePageThreeContentSeven' : "Do not access your wallet when using public "
+          "Wi-Fi or someone else’s device.",
+      'welcomePageThreeButtonBack' : 'Go Back',
+      'welcomePageThreeButtonNext' : 'I Understand',
+
+      'startPageAppBarTitle' : 'Omni Wallet',
```
Comment (achamely) [inline lib/l10n/WalletLocalizations.dart]: The format for displaying/referencing is `Omniwallet`

File: lib/l10n/WalletLocalizations.dart [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+      'backup_index_laterbackup': 'Backup later',
+      'backup_index_btn': 'Backup wallet mnemonic',
+      'backup_index_tips': 'Note: please back up your Wallet account, Omni Wallet does not access your account, cannot restore private key, reset password. You are in control of your own wallet and asset security.',
+      'backup_index_prompt_tips': 'Anyone who gets your mnemonic will get your assets. Please copy on the paper for safekeeping.',
+      'backup_index_prompt_title': "Don't screenshots",
+      'backup_words_title': 'Back up your mnemonic',
+      'backup_words_next': 'next',
+      'backup_words_content': "Please copy the mnemonic word carefully,\nwe will verify it in the next step.",
+      'backup_words_order_title': 'Confirm mnemonic words',
+      'backup_words_order_content': 'Please click mnemonic word in order to \nmake sure that you backup correctly',
```
Comment (achamely) [inline lib/l10n/WalletLocalizations.dart]: `word` -> `words`

File: lib/l10n/WalletLocalizations.dart [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+      'backup_index_prompt_btn': 'I got it',
+      'backup_index_title': 'Backup the wallet',
+      'backup_index_laterbackup': 'Backup later',
+      'backup_index_btn': 'Backup wallet mnemonic',
+      'backup_index_tips': 'Note: please back up your Wallet account, Omni Wallet does not access your account, cannot restore private key, reset password. You are in control of your own wallet and asset security.',
+      'backup_index_prompt_tips': 'Anyone who gets your mnemonic will get your assets. Please copy on the paper for safekeeping.',
+      'backup_index_prompt_title': "Don't screenshots",
+      'backup_words_title': 'Back up your mnemonic',
+      'backup_words_next': 'next',
+      'backup_words_content': "Please copy the mnemonic word carefully,\nwe will verify it in the next step.",
```
Comment (achamely) [inline lib/l10n/WalletLocalizations.dart]: It looks like you might be displaying more than 1 word at a time, if this is the case then this should be pluralized. 
`word` -> `words`
`verify it` -> `verify them`

File: lib/l10n/WalletLocalizations.dart [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+
+    'en': {
+      'main_index_title': 'LunarX_Omni Wallet',
+      'backup_index_prompt_btn': 'I got it',
+      'backup_index_title': 'Backup the wallet',
+      'backup_index_laterbackup': 'Backup later',
+      'backup_index_btn': 'Backup wallet mnemonic',
+      'backup_index_tips': 'Note: please back up your Wallet account, Omni Wallet does not access your account, cannot restore private key, reset password. You are in control of your own wallet and asset security.',
+      'backup_index_prompt_tips': 'Anyone who gets your mnemonic will get your assets. Please copy on the paper for safekeeping.',
+      'backup_index_prompt_title': "Don't screenshots",
```
Comment (achamely) [inline lib/l10n/WalletLocalizations.dart]: Do Not Use Screenshots!

File: lib/l10n/WalletLocalizations.dart [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+    },
+
+    'en': {
+      'main_index_title': 'LunarX_Omni Wallet',
+      'backup_index_prompt_btn': 'I got it',
+      'backup_index_title': 'Backup the wallet',
+      'backup_index_laterbackup': 'Backup later',
+      'backup_index_btn': 'Backup wallet mnemonic',
+      'backup_index_tips': 'Note: please back up your Wallet account, Omni Wallet does not access your account, cannot restore private key, reset password. You are in control of your own wallet and asset security.',
+      'backup_index_prompt_tips': 'Anyone who gets your mnemonic will get your assets. Please copy on the paper for safekeeping.',
```
Comment (achamely) [inline lib/l10n/WalletLocalizations.dart]: Anyone who gets access to your mnemonic will have access to your assets. Please copy it onto paper and store securely for safekeeping.

File: lib/l10n/WalletLocalizations.dart [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+      'common_btn_skip' : '跳过',
+    },
+
+    'en': {
+      'main_index_title': 'LunarX_Omni Wallet',
+      'backup_index_prompt_btn': 'I got it',
+      'backup_index_title': 'Backup the wallet',
+      'backup_index_laterbackup': 'Backup later',
+      'backup_index_btn': 'Backup wallet mnemonic',
+      'backup_index_tips': 'Note: please back up your Wallet account, Omni Wallet does not access your account, cannot restore private key, reset password. You are in control of your own wallet and asset security.',
```
Comment (achamely) [inline lib/l10n/WalletLocalizations.dart]: Note: please backup your Wallet account. Omniwallet does not access your account, cannot restore any private keys, or reset your password. You are in control of your own wallet and asset security.

File: lib/l10n/WalletLocalizations.dart [hunk 8]
```diff
@@ -0,0 +1,201 @@
...
+      'startPageButtonFirst' : '      开始使用      ',
+      'startPageButtonSecond' : '      恢复钱包      ',
+      'startPageLanguageBarTitle' : '语言',
+
+      'main_page_title' : '钱包',
+      'common_btn_skip' : '跳过',
+    },
+
+    'en': {
+      'main_index_title': 'LunarX_Omni Wallet',
```
Comment (achamely) [inline lib/l10n/WalletLocalizations.dart]: `LunarX Omniwallet`
or
`Omniwallet by LunarX`

File: lib/view/backupwallet/backup_wallet_index.dart (hunk 1 above)
Comment (mobilecheng) [reply lib/view/backupwallet/backup_wallet_index.dart]: Fixed.

File: lib/l10n/WalletLocalizations.dart (hunk 2 above)
Comment (mobilecheng) [reply lib/l10n/WalletLocalizations.dart]: Fixed.

File: lib/l10n/WalletLocalizations.dart (hunk 3 above)
Comment (mobilecheng) [reply lib/l10n/WalletLocalizations.dart]: Fixed.

File: lib/l10n/WalletLocalizations.dart (hunk 4 above)
Comment (mobilecheng) [reply lib/l10n/WalletLocalizations.dart]: Fixed.

File: lib/l10n/WalletLocalizations.dart (hunk 5 above)
Comment (mobilecheng) [reply lib/l10n/WalletLocalizations.dart]: Fixed.

File: lib/l10n/WalletLocalizations.dart (hunk 6 above)
Comment (mobilecheng) [reply lib/l10n/WalletLocalizations.dart]: Fixed.

File: lib/l10n/WalletLocalizations.dart (hunk 7 above)
Comment (mobilecheng) [reply lib/l10n/WalletLocalizations.dart]: Fixed.

File: lib/l10n/WalletLocalizations.dart (hunk 8 above)
Comment (mobilecheng) [reply lib/l10n/WalletLocalizations.dart]: Fixed.

**Pull request merged**